### ISSUE TITLE
Air crescent

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6684]
**Preferred term label:**
Air crescent

**Synonyms**

**Definition (free text, please give PubMed ID)**
An air crescent is a collection of air in a crescentic shape that separates the wall of a cavity from an inner mass. The
air crescent sign is often considered characteristic of either Aspergillus colonization of preexisting cavities or retraction of infarcted lung in angioinvasive aspergillosis. However, the air crescent sign has also been reported in other conditions, including tuberculosis, Wegener granulomatosis, intracavitary hemorrhage, and lung cancer. 

PMID:18195376

Comment
See Figure 3 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
